rdr_client/export_tables.py

```diff
@@ -0,0 +1,34 @@
+# Exports the entire contents of database tables to Unicode CSV files stored in GCS.
+# Used instead of Cloud SQL export because it handles newlines and null characters properly.
+#
+# Usage: ./run_client.sh --project <PROJECT> --account <ACCOUNT> \
+# --service_account exporter@<PROJECT>.iam.gserviceaccount.com export_tables.py \
+# --database rdr --tables code,participant --directory test_directory
+# If "rdr" is chosen for the database, the data will be written to <ENVIRONMENT>-rdr-export;
+# If "cdm" or "voc" are chosen, the data will be written to <ENVIRONMENT>-cdm.
```

Comment:
Please document the meaning of the `directory` parameter (i.e. a folder within the export bucket).

Reply:
Done.